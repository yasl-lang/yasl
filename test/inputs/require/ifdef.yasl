const m = require('test/inputs/args.yam')

echo m

ifdef args {
    echo 'script: args is defined'
} else {
    echo 'script: args is not defined'
}
