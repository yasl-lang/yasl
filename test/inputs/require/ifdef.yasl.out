submodule: args is not defined
script: args is not defined
